src/seedu/addressbook/data/person/sortByPerson.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+import java.util.*;
+
+class sortByPerson implements Comparator<Person> {
+    public int compare(Person o1, Person o2) {
+        String o1Name = o1.getName().toString();
+        String o2Name = o2.getName().toString();
+
+        if (o1Name.compareTo(o2Name) < 0) {
+            return -1;
+        }
+        else if (o1Name.compareTo(o2Name) > 0) {
```

Comment:
coding standard violation here

Reply:
Hi I don't quite get why this is a coding standard violation- read through the coding style guide and I don't spot my mistake.